### DISCUSSION TITLE
Setting to Adjust the Font Size

### DISCUSSION THREAD

**Discussion by Anthropos888** [lawand-dot-io/taskbar#45] [Feature requests]
Please add a setting to adjust the font size of the tabs.
On my 4K screen, die Font size is way too big and way too bold.

I prefer a thin and small font like on Windows 10/11

**Comment by lawand-dot-io** [comment]
This is a duplicate of https://github.com/lawand-dot-io/taskbar/discussions/36

I agree that the font size and the taskbar size in should be configurable :)
